### PR TITLE
Number of patients registered today.

### LINKED ISSUE #72: Display number of patients registered today

### PR DESCRIPTION
**Pull request by erictzeng**
Resolves #72.

If someone knows of a nicer way to handle "1 patient" vs. "2 patients" in the view, I'm all ears!

### CODE REVIEW

File: app/views/registrars/_rightbar.html.haml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 %h3 At a glance
 %div.stat
-  %div.bignum 23
-  patients registered today
+  %div.bignum= @num_today
```
Comment (danwang) [inline app/views/registrars/_rightbar.html.haml:3]: I did the average time per patient by passing in the `Registrar` object. We should probably do one way or another for consistency. Why did you choose to pass in `num_today` explicitly?

File: app/models/registrar.rb [hunk 2]
```diff
@@ -2,4 +2,7 @@ class Registrar < ActiveRecord::Base
   attr_accessible :name
   validates :name, :presence => true
   has_many :registrations
+  def registrations_for_day(day)
```
Comment (danwang) [inline app/models/registrar.rb:5]: This seems unnecessary

Comment (erictzeng) [inline app/models/registrar.rb:5]: Law of Demeter, mang

File: app/views/registrars/_rightbar.html.haml (hunk 1 above)
Comment (danwang) [inline app/views/registrars/_rightbar.html.haml:3]: This makes sense with the comments on #77.

Comment (erictzeng) [inline app/views/registrars/_rightbar.html.haml:3]: Because there is no `Registrar` object for the overall view. (`/registration/registrars`)

**Comment by danwang** [comment]
http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-pluralize

File: app/models/registrar.rb (hunk 2 above)
Comment (erictzeng) [inline app/models/registrar.rb:5]: http://www.lukeredpath.co.uk/blog/demeters-revenge.html

**Comment by danwang** [comment]
Good ya

**Pull request merged**